Tropos/Models/DailyForecast.swift

```diff
@@ -0,0 +1,15 @@
+import Foundation
+
+@objc class DailyForecast: NSObject {
```

Comment:
Great, I legitimately wasn't sure. We should do this going forward. Making sure we're not subclassing will make the transition to structs easier later.